### PR TITLE
Clear `forked-to` cache after `quick-repo-deletion`

### PR DESCRIPTION
**Pull request by kidonng**
I usually try not to correlate features together, but this is a convenient and harmless improvement we can do.

## Test URLs

1. Fork https://github.com/octocat/Spoon-Knife
2. Visit your fork
3. Delete the fork using `quick-repo-deletion`
4. Return to https://github.com/octocat/Spoon-Knife, `forked-to` should not be present

## Screenshot

N/A

### CODE REVIEW

File: source/features/quick-repo-deletion.tsx [hunk 1]
```diff
@@ -103,6 +105,7 @@ async function start(buttonContainer: HTMLDetailsElement): Promise<void> {
 			{action: false},
 		);
 		select('.application-main')!.remove();
+		await cache.delete(getCacheKey());
```
Comment (cheap-glitch) [inline source/features/quick-repo-deletion.tsx:108]: Could this throw an error if the key doesn't exist in the cache? The user might have disabled `forked-to` for example.

**Review by cheap-glitch — COMMENTED** [review]
> I usually try to correlate features

I think you mean "decorrelate"? :smiley: 

So if I understand correctly, the point of this is to avoid showing outdated info in `forked-to`?

**Comment by kidonng** [comment]
> I think you mean "decorrelate"? :smiley: 
> 

Good catch, I missed one `not` there.

> So if I understand correctly, the point of this is to avoid showing outdated info in `forked-to`?

Right, even though the expire time is pretty short, it still looks like a glitch that should have been handled.

File: source/features/quick-repo-deletion.tsx (hunk 1 above)
Comment (kidonng) [reply source/features/quick-repo-deletion.tsx:108]: I've tested this beforehand and it does nothing if there's no such key.

**Review by fregante — APPROVED** [review]

**Pull request merged**